homework-23/task1_person.py

```diff
@@ -0,0 +1,16 @@
+class Person:
+    def __init__(self, name, age):
+        self._name = name
+        self._age = age
```

Comment:
როგორც ამ დავალებაში ისე დანარჩენ ორშიც კარგი პრაქტიკა იქნება თუ @Property დეკორატორს გამოვიყენებთ.
```suggestion

    @property
    def name(self):
        return self._name

    @name.setter
    def name(self, _name):
        if not _name:
            raise ValueError("Name cannot be empty.")
        self._name = _name

    @property
    def age(self):
        return self._age

    @age.setter
    def age(self, _age):
        if not isinstance(_age, int) or _age <= 0:
            raise ValueError("Age must be a positive integer.")
        self._age = _age

    def get_info(self):
        return f"Name: {self.name}, Age: {self.age}"

```